src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -372,6 +374,15 @@ CLParser::Usage(const std::string& msg)
              "that the average latency is used to determine stability",
              18)
       << std::endl;
+  std::cerr << FormatMessage(
+                   " --overhead-percentage-threshold: The percentage of total "
```

Comment:
This description could use some work. Follow this format:
`--arg_name: What the arg is. More details.`

Your first sentence is NOT what the arg is. It is defining what overhead percentage is (without telling the user that is what you are defining).

Reply:
Updated the description to better reflect what this is.